src/test/functions/listFunctions.spec.ts

```diff
@@ -0,0 +1,160 @@
+import { expect } from "chai";
+import * as sinon from "sinon";
+import { listFunctions } from "../../functions/listFunctions";
+import * as backend from "../../deploy/functions/backend";
+import * as args from "../../deploy/functions/args";
+import { previews } from "../../previews";
+
+describe("listFunctions", () => {
```

Comment:
Is nesting `describe` under another `describe` intentional here?

Reply:
Yeah for the filename and function. I guess it's kind of confusing since they're both the same name.

Reply:
I removed this set of unit tests and added test cases to compareFunction